### ISSUE TITLE
TrimMultilineRawString support for alternate trimming methods

### ISSUE THREAD

**Issue by samuelneff** [detekt/detekt#5415]
The `TrimMultilineRawString` (https://github.com/detekt/detekt/pull/5051) rule can be helpfully extended to allow custom trimming functions and chained functions. 

## Expected Behavior
Here is an example from our codebase:

```kotlin
fun String.collapseWhitespace(text: String) =
    trim().replace(Regex("\\s\\s+"), " ")

val blockWithoutAnyExtraWhitespace = """
    This is some text.

    Here is another line; spaced apart.
""".collapseWhitespace()

println(blockWithoutAnyExtraWhitespace)
// Prints: This is some text. Here is another line; spaced apart.
```

## Current Behavior

Currently the rule requires `trimIndent()` or `trimMargin()` be called and nothing else is considered valid.

## Context

See above example from our codebase that fails this rule but we would like to be able to pass. 
Configuration with this change would include an additional option:

```yml
TrimMultilineRawString:
  additionalTrimmingMethods:
    - collapseWhitespace
```

Note: I'm working on another rule now for Hacktoberfest. If I actually get one done, I'll look into implementing this as well. Looks straightforward. Thanks for this great new rule!

**Comment by BraisGabin** [comment]
Good idea! But instead of `additionalTrimmimgMethods` I would call it `trimmingMethods` and add the two default ones.

Do you want me to assign this issue to you?

**Comment by samuelneff** [comment]
> `trimmingMethods`

:+1:

> assign this issue to you

hesitant :+1: 😄

**Comment by BraisGabin** [comment]
Hahaha, no pressure ;)
